sdk/src/rest/utils/query-to-params.ts

```diff
@@ -0,0 +1,12 @@
+import type { Query } from '../../types/index.js';
+
+export const queryToParams = (query: Query<any>): Record<string, string> => {
+	const params: Record<string, string> = {};
+
+	if (query.fields) {
```

Comment:
This obv also needs the params for the other values.

Note: the API accepts JSON for most params, so instead of overthinking `filter`, just set `params['filter'] = JSON.stringify(query.filter)`